qiskit/algorithms/state_fidelities/__init__.py

```diff
@@ -0,0 +1,41 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+=====================================================================
+State Fidelity Interfaces (:mod:`qiskit.algorithms.state_fidelities`)
+=====================================================================
+
+.. currentmodule:: qiskit.algorithms.state_fidelities
+
+State Fidelities
+================
+
+.. autosummary::
+   :toctree: ../stubs/
+
+   BaseStateFidelity
+   ComputeUncompute
+
+Results
+=======
+
+ .. autosummary::
+    :toctree: ../stubs/
+
+    StateFidelityResult
```

Comment:
```suggestion
    StateFidelityResult

```
I think this extra blank line is the reason the formatting on the resultant page the result here does not span the full width